commands/pack.js

```diff
@@ -47,6 +47,7 @@ function displayHelp() {
 	console.log('                      (requires V8_MKSNAPSHOT set)');
 	console.log('    -m, --meta        JSON to override package.json enact metadata');
 	console.log('    -c, --custom-skin Build with a custom skin');
+	console.log('    --content-hash    Add the hash of the chunk');
```

Comment:
```suggestion
	console.log('    --content-hash    Add a unique hash to output files based on the content of an asset');
```

Reply:
Let's add `-h` for shorthand.

Reply:
And move this option order to after `output`.

Reply:
OK